deep-learning/_posts/2019-05-06-ENet.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: review
+title:  "ENet: A Deep Neural Network Architecture for Real-Time Semantic Segmentation"
+tags:   deep-learning, segmentation, benchmarking, CNN
```

Comment:
Tags need not to (must not?) be comma-separated, whitespace separation is enough, right?

Reply:
Done in a037b83.